src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -54,12 +54,21 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the appointment list with {@code appointments}.
+     * {@code persons} must not contain duplicate persons.
```

Comment:
I think we can change the comment here to appointments so it's more relevant

Reply:
Thanks for spotting this! Making changes now.